server/controllers/list.js

```diff
@@ -111,18 +113,31 @@ const getArchivedListsMetaData = (req, resp) => {
   );
 };
 
+const checkIfCurrentUserVoted = (item, userId) =>
```

Comment:
this should not have in name `current` if you pass `userId` as param. When thinking about function name try to use it in `if` statement and see how it reads. Currently it sound strange `if (checkIfCurrentUserVoted(...))`. Maybe just `if (userVoted(..))`?